build-libs/redirects.js

```diff
@@ -52,12 +53,28 @@ const HOSTNAME_MAP = {
  * @param {string} redirectsPath Path within the repo to the redirects file.
  * @returns {Promise<Redirect[]>}
  */
-async function getRedirectsFromContentRepo(repoName, redirectsPath) {
+async function getRedirectsFromContentRepo(repoName, redirectsPath, config) {
 	/**
 	 * Note: These constants are declared for clarity in build context intent.
 	 */
 	const isDeveloperBuild = !process.env.IS_CONTENT_PREVIEW
 	const isLocalContentBuild = isDeployPreview(repoName)
+	/**
+	 * Load redirects from the unified docs repo if it's in the list of migrated repos.
+	 * Return early if there are not any redirects found for that specific repo.
+	 */
+	if (
+		config.flags?.unified_docs_migrated_repos?.find((repo) => repo === repoName)
+	) {
+		const getUDRRedirects = await fetch(
+			`${process.env.UNIFIED_DOCS_API}/api/content/${repoName}/redirects`
+		)
+		if (getUDRRedirects.ok) {
+			const udrRedirects = await getUDRRedirects.json()
+			return udrRedirects
+		}
```

Comment:
@LeahMarieBush If we fail to get a good response for the redirects API we should at least console.error. (If not fail the build, but I'm not sure about failing the build yet. I need to think through that case...)